sass/style.sass

```diff
@@ -24,71 +26,249 @@ body
 .cards-wrapper
-  margin-bottom: 20px
+  margin-bottom: $px20
   width: 100%
   display: flex
   justify-content: space-between
 
 
 .portfolio-button
-  border: none
-  padding: 8px
-  min-width: 60px
-  vertical-align: middle
-  text-align: center
-  cursor: pointer
-  background-color: white
-
-  &:hover
-    background-color: $mddl-slvr
-    color: black
+  @include base-btn($black, $white)
+  @include hover-style($black, $grey-md)
 
-  &--active
-    @extend .portfolio-button
-    color: white
-    background-color: black
+  &--first
+    @include base-btn($white, $black)
+    @include hover-style($black, $grey-md)
 
   &--footer
-    @extend .portfolio-button
-    color: black
-    background-color: inherit
-    &:hover
-      @extend .portfolio-button--active
+    @include base-btn($black, $bg-dflt-clr)
+    @include hover-style($white, $black)
+
+    &--first
+      @extend .portfolio-button--first
+
+  &--color
+    @include base-btn($white, $green)
+    @include hover-style($white, $black)
+    width: 50%
+    height: 50%
+    font-size: 20px
+
+  &--download
+    @include base-btn($white, $grey-hard)
+    @include hover-style($black, $grey-md)
+    min-width: 20%
+    height: 50px
+    padding: 0 $px20
+    margin-top: $px20
+    font-size: 18px
+
 
 .item
   width: 32%
   height: 20%
-  background-color: $bg-wht
+  background-color: $white
 
   img
     width: 100%
-    height: 400px
+    height: 300px
     object-fit: cover
+
     &:hover
       opacity: 70%
 
   .item-container
     height: 33%
-    padding: 10px 20px
+    padding: 10px $px20
+
     p
       display: -webkit-box
       height: 100%
       -webkit-line-clamp: 4
       -webkit-box-orient: vertical
       overflow: hidden
       text-overflow: ellipsis
-      line-height: 20px
+      line-height: $px20
+
+
+.charge-wrapper
+  display: grid
+  grid-template-columns: 1fr 1fr 1fr
+  grid-gap: 10px
+
+.charge-item
+  width: 100%
+  height: 400px
+  background-color: $white
+  display: grid
+  grid-template-rows: 20% 10% 10% 10% 10% 20% 20%
+  border: $grey-md solid 1px
+  opacity: 50%
+
+  &:hover
+    opacity: 100%
+
+  .charge-item-cell
+    border-bottom: $grey-md solid 1px
+    display: flex
+    flex-direction: column
+    align-items: center
+    justify-content: center
+
+    h1
+      margin: 0
+
+    &:last-child
+      border-bottom: 0px
+      background-color: $bg-dflt-clr
+
+    &:first-child
+      background-color: $black
+      color: $white
+
+  &:nth-child(2n)
+    div:first-child
+      background-color: $green
+      color: $white
+
+.about-wrapper
+  margin: 0
+
+  h1, p
+    margin-bottom: $px20
+
+.about-images
+  display: flex
+  justify-content: space-between
+  flex-direction: row
+  margin-bottom: $px20
+
+  img
+    width: 49%
+    height: 300px
+    object-fit: cover
 
+
+.skills-wrapper
+  margin: 40px 0
+
+  h3, div
+    margin-bottom: 10px
+
+.scale
+  width: 100%
+  height: 40px
+  background-color: $grey-light
+  color: $white
+
+.percent
```

Comment:
You could use `percent` class only
`.percent:first-child`
`.percent:nth-child(2)`
`.percent:last-child`